site/app/controllers/HomePageController.php

```diff
@@ -135,21 +136,68 @@ function(Course $course) {
      * @return Response
      */
     public function showHomepage() {
-        $courses = $this->getCourses()->json_response->json;
+        $courses = $this->getCourses()->json_response->json;        
```

Comment:
Why is this line different?

Reply:
"git add -p" quirks, most likely

Reply:
My guess is there is a white space difference. Please check to make sure formatting is the same. The project has a defined style guideline. Make sure the PR follows it.